src/config_loader.coffee

```diff
@@ -1,15 +1,23 @@
 fs = require 'fs'
 path = require 'path'
+yaml = require 'js-yaml'
+
+
+parseConfig = (contents, done) ->
+  try
+    config = yaml.safeLoad contents
+  catch err
+    return done err
+  done null, config
```

Comment:
If you put the done in the try, then it has the possibly of catching errors thrown in done, which isn't something I want to do